dts/bindings/sensor/st,lps22df-common.yaml

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+include: sensor-device.yaml
+
+properties:
+  drdy-gpios:
+    type: phandle-array
+    description: |
+      DRDY pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  drdy-pulsed:
+    type: boolean
+    description: |
+      Selects the pulsed mode for data-ready interrupt when enabled,
+      and the latched mode when disabled.
+
+  odr:
+    type: int
+    default: 0
+    description: |
+        Specify the output data rate expressed in samples per second (Hz).
+        The default is the power-on reset value.
+    enum:
+      - 0  # Power-Down
+      - 1  # 1Hz
+      - 2  # 4Hz
+      - 3  # 10Hz
+      - 4  # 25Hz
+      - 5  # 50Hz
+      - 6  # 75Hz
+      - 7  # 100Hz
+      - 8  # 200Hz
```

Comment:
Description need to be updated here as well

Reply:
NOt sure why i'm so stupid